snippets/markdown.json

```
@@ -249,5 +249,32 @@
 				"   {{< nextlink href=\"link/here\" >}}Link desctiption goes here{{< /nextlink >}}",
 				"{{< /whatsnext >}}"
 			]
+	},
+	"Eng Blog Frontmatter":{
+		"prefix": ";;eng_blog_frontmatter",
+		"body": [
+			"---",
+			"authors:",
+			"- email: author@datadoghq.com",
+			"  name: author-full-name",
+			"  image: author-headshot-image.jpg",
+			"blog: engineering",
+			"date: YYYY-MM-DD",
+			"publishDate: YYYY-MM-DD",
+			"description: \"A-short-summary-of-the-post-to-be-used-on-the-front-page-of-the-blog\"",
```

```suggestion
			"description: \"A short summary of the post to be used on the front page of the blog\"",
```
Just because some people will match the exact format of whatever's given to them, and I don't think you actually wanted the dashes in the final product.